src/client/opamArg.ml

```diff
@@ -1263,8 +1263,10 @@ let global_options cli =
   let cudf_file =
     mk_opt ~cli cli_original ~section ["cudf"] "FILENAME"
       "Debug option: Save the CUDF requests sent to the solver to \
-       $(docv)-<n>.cudf."
-      Arg.(some string) None in
+       $(docv)-<n>.cudf where <n> is the n-th call to the solver during \
+       the opam run call of opam."
```

Comment:
```suggestion
       the current call of opam."
```

Reply:
oups, forgot to remove some words, i wanted to replace with `the opam run`